scss/_patterns_grid.scss

```diff
@@ -27,44 +27,21 @@
 }
 
 @mixin vf-p-grid {
-  %span-full-grid-on-mobile {
-    @media (max-width: $threshold-4-6-col) {
-      grid-column: 1 / span $grid-columns-small;
-    }
-  }
-
-  %span-full-grid-on-tablet {
-    @media (min-width: $threshold-4-6-col) and (max-width: $threshold-6-12-col) {
-      grid-column: 1 / span $grid-columns-medium;
-    }
-  }
-
-  %span-full-grid-on-desktop {
-    @media (min-width: $threshold-6-12-col) {
-      grid-column: 1 / span $grid-columns;
-    }
-  }
-
   // FIXME: this should be removed from framework SCSS
   // (see https://github.com/canonical-web-and-design/vanilla-framework/issues/3199)
   .grid-demo [class*='#{$grid-column-prefix}'] {
     background: transparentize($color-negative, 0.9);
     margin-bottom: $spv-outer--small;
   }
 
-  %display-block {
-    // make columns explicitly display:block; the use of a placeholder is to ensure the rule appears only once in the compiled css
-    display: block;
-  }
-
   .row {
     @extend %vf-row;
   }
 
   // mobile grid
   @media (max-width: $threshold-4-6-col) {
     @for $i from $grid-columns-small through 1 {
-      .#{$grid-small-col-prefix}#{$i} {
+      .row .#{$grid-small-col-prefix}#{$i} {
```

Comment:
I guess the reason to add `.row` here is to make sure `col-X` class names work only in standard row context but not on new breakout layout items?

Reply:
yes, otherwise it will have unpredictable effect. Just a driveby bug fix.